documentation/alert-runbook.md

```diff
@@ -0,0 +1,41 @@
+# Alert Runbook
+
+## ProdRequestsFailuresElevated
+
+Alerts when `teaching-vacancies-production` app has more than 25 failed requests for a 5-minute period.
+Failed requests are defined as those with HTTP status codes matching the patterns `0xx`, `4xx` or `5xx`.
+
+## ProdCPUHigh
+
+Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-worker-production` app has an average CPU usage above 60% for a 5-minute period.
+
+## ProdDiskUtilisationHigh
+
+Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-worker-production` app has an average Disk utilization above 60% for a 5-minute period.
+
+## ProdMemoryUtilizationHigh
+
+Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-worker-production` app has an average Memory utilization above 60% for a 5-minute period.
+
+## ProdAppsCrashed
+
+Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-worker-production` app reports a crash.
+
+## Action
+
+[Check the Grafana panel for an indication of what's going on.](https://grafana-teaching-vacancies.london.cloudapps.digital/d/6Ac4lUWGk/teaching-vacancies-production?orgId=1&refresh=5s)
+
+### CloudFoundry
+
+Further information on setting up and logging in to CloudFoundry are in the [hosting](../hosting.md) document.
+
+- List apps with `cf apps`.
+- Check if any apps are crashed (e.g. `processes` may list 3 of 4 apps running: `web:3/4`)
+- Get the logs for the app
+```bash
+cf logs teaching-vacancies-production
+```
```

Comment:
Can you suggest:
- restart app
- scale up number of instances, memory, disk...